scripts/docker_test.sh

```diff
@@ -2,4 +2,4 @@
 
 docker run -t --rm --name test-runner-opentracing-javascript \
```

Comment:
I cannot answer that @yurishkuro, I didn't write this. I just adapted it to not use the Makefile